firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -145,20 +151,31 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
         guard let defaultFooter = _defaultFooter else { return nil }
 
         if section == 0 {
-            // TODO: Get a dedicated string for this.
-            let title: String = .TrackerProtectionLearnMore
+            let linkButtonViewModel = LinkButtonViewModel(
+                title: .TrackerProtectionLearnMore,
+                a11yIdentifier: AccessibilityIdentifiers.Settings.ContentBlocker.title,
+                font: FXFontStyles.Regular.caption1.scaledFont()
+            )
+            linkButton.configure(viewModel: linkButtonViewModel)
 
             let theme = themeManager.currentTheme(for: windowUUID)
-            let font = FXFontStyles.Regular.caption1.scaledFont()
-            var attributes = [NSAttributedString.Key: AnyObject]()
-            attributes[NSAttributedString.Key.foregroundColor] = theme.colors.actionPrimary
-            attributes[NSAttributedString.Key.font] = font
+            linkButton.applyTheme(theme: theme)
+
+            linkButton.addTarget(self, action: #selector(moreInfoTapped), for: .touchUpInside)
+            linkButton.isHidden = false
 
-            button.setAttributedTitle(NSAttributedString(string: title, attributes: attributes), for: .normal)
-            button.addTarget(self, action: #selector(moreInfoTapped), for: .touchUpInside)
-            button.isHidden = false
+            defaultFooter.contentView.addSubview(linkButton)
 
-            defaultFooter.stackView.addArrangedSubview(button)
```

Comment:
I did this change because the text and the button wouldn’t align correctly even if I tried to add constraints. If you consider that I should keep it how it was initially, please let me know. Thanks 🙏🏻

Reply:
What about adjusting the `contentInsets` in the `LinkButtonViewModel` initializer, so that the leading and trailing is `0`? Let me know if that works!